src/storage/oos_file.cpp

```diff
@@ -0,0 +1,600 @@
+/*
+ * Copyright 2008 Search Solution Corporation
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+#include <cassert>
+#include "error_code.h"
+#include "file_manager.h"
+#include "memory_alloc.h"
+#include "page_buffer.h"
+#include "porting_inline.hpp"
+#include "scope_exit.hpp"
+#include "slotted_page.h"
+#include "storage_common.h"
+#include "page_buffer_util.hpp"
+
+#include "oos_file.hpp"
+#include "oos_log.hpp"
+// XXX: SHOULD BE THE LAST INCLUDE HEADER
+#include "memory_wrapper.hpp"
+
+struct oos_record_header
+{
+  int total_size;
+  int chunk_index;
+  OID next_chunk_oid;
+};
+using OOS_RECORD_HEADER = struct oos_record_header;
+
+// ****************************************************************************
+// static functions
+// ****************************************************************************
+
+static const auto_unfix_page_ptr
+oos_find_best_page (THREAD_ENTRY *thread_p, const VFID &oos_vfid, const int rec_length, VPID &vpid);
+
+static int
+oos_vpid_init_new (THREAD_ENTRY *thread_p, PAGE_PTR page, void *args);
+
+static int oos_insert_within_page (THREAD_ENTRY *thread_p, const VFID &oos_vfid, RECDES &recdes,
```

Comment:
모든 함수를 아래와 같이 CUBRID 스타일로 변경해 주세요. (코드 일관성)
```suggestion
static int
oos_insert_within_page (THREAD_ENTRY *thread_p, const VFID &oos_vfid, RECDES &recdes,
```

Reply:
변경했습니다.
<img width="1658" height="1139" alt="image" src="https://github.com/user-attachments/assets/fd39f302-8ef3-4ec2-98c1-c98e59429c1d" />

Reply:
NIT: 코드 스타일 툴에 걸리지 않으면 문제는 안돼기 때문에 참고하셔서 추가 체크 및 반영 검토해주세요.
1. oos_file_destroy(), oos_insert(), get_recently_inserted_oos_vpid() 등 아직 반영되지 않은 함수들이 보입니다.
2. heap_file.c 등을 참고해 보았을 때 이유는 모르겠으나 다음과 같은 규칙이 적용되어 있습니다.
```
// 선언부
static int oos_read_within_page (...);

// 구현부
static int
oos_read_within_page (...)
```

Reply:
함수 이름 전수조사하여 변경 완료했습니다. 감사합니다.
<img width="1861" height="1311" alt="image" src="https://github.com/user-attachments/assets/61eeedfa-646c-420e-8913-39b0c2c94278" />